packages/fast-usdc/src/fast-usdc.contract.ts

```diff
@@ -8,65 +8,62 @@ import { observeIteration, subscribeEach } from '@agoric/notifier';
 import {
   OrchestrationPowersShape,
   withOrchestration,
+  type Denom,
+  type OrchestrationAccount,
+  type OrchestrationPowers,
+  type OrchestrationTools,
 } from '@agoric/orchestration';
+import { makeZoeTools } from '@agoric/orchestration/src/utils/zoe-tools.js';
 import { provideSingleton } from '@agoric/zoe/src/contractSupport/durability.js';
 import { prepareRecorderKitMakers } from '@agoric/zoe/src/contractSupport/recorder.js';
-import { makeZoeTools } from '@agoric/orchestration/src/utils/zoe-tools.js';
 import { depositToSeat } from '@agoric/zoe/src/contractSupport/zoeHelpers.js';
 import { E } from '@endo/far';
 import { M, objectMap } from '@endo/patterns';
+import type { Zone } from '@agoric/zone';
```

Comment:
I was concerned that conversion to .ts would mean that type imports have to be changed from devDependencies to dependencies, but I don't see a change to `package.json` for `@agoric/zone`, so evidently not. Interesting.